### PR TITLE
Epic login component

### PR DESCRIPTION
**Pull request by lmiotti**
### Summary
Wolox logo and cover added, email and password edit text added, Sign Up and Log in buttons added and Terms and conditions link to google site

### Screenshot
image

### Trello Card
https://trello.com/c/bjzWlmHG/18-u1-frontend-login-componentes-b%C3%A1sicos

### CODE REVIEW

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:padding="1dp"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_login.xml]: We should send all the dimens to dimens.xml file :)

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            android:src="@drawable/wolox_logo_large" />
+
+    </android.support.constraint.ConstraintLayout>
+
+    <EditText
+        android:id="@+id/fragment_login_email_address_edit_text"
+        style="@style/EditTextTheme"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="64dp"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_login.xml]: Same here with dimens

File: app/src/main/res/layout/fragment_login.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        android:textSize="18sp"
+        android:text="@string/login_email_address" />
+
+    <EditText
+        android:id="@+id/fragment_login_password_edit_text"
+        style="@style/EditTextTheme"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_login.xml]: More dimens :3

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 
     <!-- General -->
-    <string name="example_title">ExampleFragment</string>
-    <string name="example_textview">Some example TextView</string>
-    <string name="example_randomize">Randomize</string>
-    <string name="example_wolox">WOLOX</string>
-    <string name="example_message">The presenter generated the number %1$d and this view is displaying it</string>
+    <string name="login_email_address">Email address</string>
+    <string name="login_password">Password</string>
+    <string name="login_login">LOG IN</string>
```
Comment (DemianIac) [inline app/src/main/res/values/strings.xml:6]: You can write the text like Log in and then set the uppercase to the button . Then you can use the same string resource in other components :)

**Review by DemianIac — CHANGES_REQUESTED** [review]
Just a few minor changes . Great work!

**Comment by lmiotti** [comment]
### Summary
Dimens.xml file was created, and dimensions in fragment_layout was referenced to that file.

### Screenshot

image

### Trello Card

https://trello.com/c/bjzWlmHG/18-u1-frontend-login-componentes-b%C3%A1sicos

**Review by DemianIac — APPROVED** [review]
:+1:

**Pull request merged**